pkgs/development/python-modules/meraki/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python3
+}:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
+  pname = "meraki";
+  version = "1.30.0";
```

I think we're starting to prefer adding `format` by default for all python packages, so it's probably best to make that explicit here:

```suggestion
  version = "1.30.0";
  format = "setuptools";
```